pandas/core/arrays/string_arrow.py

```diff
@@ -345,14 +345,14 @@ def _str_match(
         self, pat: str, case: bool = True, flags: int = 0, na: Scalar | None = None
     ):
         if not pat.startswith("^"):
-            pat = "^" + pat
```

Comment:
i dont think these are an improvement.  stick to places that currently use .format

Reply:
Would this apply to all string concatenation? I think e.g. the changes in `sql.py` (https://github.com/pandas-dev/pandas/pull/49229/files#diff-9268174bfb15f08ef2267375665a85fecf201999902542f6fc9c0d3fadfb4553 if GitHub feels like linking correctly) read much better as an f-string, for one?